### ISSUE TITLE
A design to install rules

### ISSUE THREAD

**Issue by gcampax** [Stanford-IoT-Lab/ThingPedia#5]
That is, some way for a user to click through a rule in ThingPedia and have it installed on ThingEngine (or at least on ThingEngine cloud).
Needs design for the API part, we had something in the summer but it was kind of broken.

Depends on issue 3 for the "show rule" UI.

Note: ThingEngine cloud service has a OAuth implemented, if we need it. The user would have to associate the accounts in some way though.
Or we get rid of login with username/password for ThingPedia and use OAuth logins from ThingEngine always...

**Comment by gcampax** [comment]
I think this is sufficiently done as of now, you can install pretty much anything just by clicking through, you can fill parameters, etc.

We'll open a new issue if we want to do better.
